lib/AST/ImportCache.cpp

```diff
@@ -55,25 +55,25 @@ void ImportSet::Profile(
     ArrayRef<ModuleDecl::ImportedModule> topLevelImports) {
   ID.AddInteger(topLevelImports.size());
   for (auto import : topLevelImports) {
-    ID.AddInteger(import.first.size());
-    for (auto accessPathElt : import.first) {
+    ID.AddInteger(import.accessPath.size());
+    for (auto accessPathElt : import.accessPath) {
       ID.AddPointer(accessPathElt.Item.getAsOpaquePointer());
     }
-    ID.AddPointer(import.second);
+    ID.AddPointer(import.importedModule);
   }
 }
 
 static void collectExports(ModuleDecl::ImportedModule next,
                            SmallVectorImpl<ModuleDecl::ImportedModule> &stack) {
   SmallVector<ModuleDecl::ImportedModule, 4> exports;
-  next.second->getImportedModulesForLookup(exports);
+  next.importedModule->getImportedModulesForLookup(exports);
   for (auto exported : exports) {
-    if (next.first.empty())
+    if (next.accessPath.empty())
       stack.push_back(exported);
-    else if (exported.first.empty()) {
-      exported.first = next.first;
+    else if (exported.accessPath.empty()) {
+      exported.accessPath = next.accessPath;
       stack.push_back(exported);
-    } else if (ModuleDecl::isSameAccessPath(next.first, exported.first)) {
+    } else if (ModuleDecl::isSameAccessPath(next.accessPath, exported.accessPath)) {
```

Comment:
I think there's a few straggling lines that are over 80 characters, and the indenting certainly gets tricky. Would you mind running clang-format (LLVM style) over your changes?

Reply:
I suspect what xwu means is using `llvm-project/clang/tools/clang-format/clang-format-diff.py`, not using clang-format directly, as that would reformat entire files.